src/ContentBlocks/TextBlock.php

```diff
@@ -0,0 +1,233 @@
+    {
+        return new self();
+    }   
+
+    /**
+     * Set the text.
+     *
+     * @param string $text
+     * 
+     * @return TextBlock
+     */
+    public function setText(string $text) : self
+    {
+        $this->text = $text;
+        return $this;
+    }
+
+    /**
+     * Set the color.
+     *
+     * @param string $color
+     * 
+     * @return TextBlock
+     */
+    public function setColor(string $color) : self
+    {
+        $this->color = $color;
+        return $this;
+    }
+
+    /**
+     * Set the wrap.
+     *
+     * @param bool $wrap
+     * 
+     * @return TextBlock
+     */
+    public function setWrap(bool $wrap) : self
+    {
+        $this->wrap = $wrap;
+        return $this;
+    }
+
+    /**
+     * Set the spacing.
+     *
+     * @param string $spacing
+     * 
+     * @return TextBlock
+     */
+    public function setSpacing(string $spacing) : self
+    {
+        $this->spacing = $spacing;
+        return $this;
+    }
+
+    /**
+     * Set the separator.
+     *
+     * @param bool $separator
+     * 
+     * @return TextBlock
+     */
+    public function setSeparator(bool $separator) : self
+    {
+        $this->separator = $separator;
+        return $this;
+    }
+
+    /**
+     * Set the font type.
+     *
+     * @param string $fontType
+     * 
+     * @return TextBlock
+     */
+    public function setFontType(string $fontType) : self
+    {
+        $this->fontType = $fontType;
+        return $this;
+    }
+
+    /**
+     * Set the font size.
+     *
+     * @param string $fontSize
+     * 
+     * @return TextBlock
+     */
+    public function setSize(string $fontSize) : self
+    {
+        $this->size = $fontSize;
+        return $this;
+    }
+
+    /**
+     * Set the weight.
+     *
+     * @param string $weight
+     * 
+     * @return TextBlock
+     */
+    public function setWeight(string $weight) : self
+    {
+        $this->weight = $weight;
+        return $this;
+    }
+
+    /**
+     * Set the is subtle.
+     *
+     * @param bool $isSubtle
+     * 
+     * @return TextBlock
+     */
+    public function setIsSubtle(bool $isSubtle) : self
+    {
+        $this->isSubtle = $isSubtle;
+        return $this;
+    }
+
+    /**
+     * Set the style.
+     *
+     * @param string $style
+     * 
+     * @return TextBlock
+     */
+    public function setStyle(string $style) : self
+    {
+        $this->style = $style;
+        return $this;
+    }
+
+    /**
+     * Set the horizontal alignment.
+     *
+     * @param string $horizontalAlignment
+     * 
+     * @return TextBlock
+     */
+    public function setHorizontalAlignment(string $horizontalAlignment) : self
+    {
+        $this->horizontalAlignment = $horizontalAlignment;
+        return $this;
+    }
+
+    /**
+     * Set the maximum lines.
+     *
+     * @param int $maxLines
+     * 
+     * @return TextBlock
+     */
+    public function setMaximumLInes(int $maxLines) : self
```

Comment:
```suggestion
    public function setMaximumLines(int $maxLines) : self
```

Reply:
resolved in https://github.com/laravel-notification-channels/microsoft-teams/pull/36/commits/0b82d791443d47e5a2c05cd2377c5e82a17a9546